checklists/alz_checklist.en.json

```diff
@@ -1006,6 +1006,78 @@
             "severity": "High",
             "link": "https://learn.microsoft.com/azure/expressroute/designing-for-high-availability-with-expressroute#active-active-connections"
         },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Hybrid",
+            "text": "Ensure the two physical links of your ExpressRoute circuit are connected to two distinct edge devices in your network.",
+            "waf": "Reliability",
+            "service": "ExpressRoute",
+            "guid": "b258f058-b9f6-46cd-b28d-990106f0c3f8",
+            "id": "D04.19",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
+            "link": "https://learn.microsoft.com/en-us/azure/expressroute/designing-for-high-availability-with-expressroute"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Hybrid",
+            "text": "Ensure Bidirectional Forwarding Detection (BFD) is enabled and configured on customer or provider edge routing devices.",
```

Comment:
Are we defaulting to this option on all scenarios? If so, maybe we should update this:

https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/connectivity-to-azure

Reply:
@igorjnzl in this PR I'm just aligning the checklist with APRL recommendations for ExpressRoute, specifically this one: https://azure.github.io/Azure-Proactive-Resiliency-Library/services/networking/expressroute-circuits/#erc-4---ensure-bidirectional-forwarding-detection-is-enabled-and-configured-on-customer-or-provider-edge-routing-devices. Then, should we follow APRL recommendation? If not, I can remove this, otherwise we can update also the CAF content. What do you think? Can you briefly discuss on the CSU side? Happy to quickly sync offline if required. Thank you.

Reply:
Let's sync offline, CSU team will reach out to discuss

Reply:
Is this still open? @igorpag

Reply:
Yes, @brsteph . @igorjnzl  promised CSU should have reached me to further discuss, but didn't happen yet, I'm still waiting.

Reply:
@igorjnzl we would need to have this out by March 27th. May I ask you to have your CSU contacts to reach me as soon as possible? Thank you.